.gitignore

```diff
@@ -10,3 +10,9 @@ dist/
 
 # Rever
 rever/
+
+# Notepad++ files
+nppBackup/
```

Comment:
This kind of editor-specific thing should be in your [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore). doesn't particularly hurt anything to have it here, but we're not ignoring `*~` or other kinds of backup files....

Reply:
Okay, thanks! For some reason I wasn't aware one could have a global ``.gitignore``, and never thought to look it up, so I appreciate the tip.